File: app/src/main/java/com/geridea/trentastico/gui/activities/IMenuSettings.java
package com.geridea.trentastico.gui.activities;


/*
 * Created with ♥ by Slava on 22/04/2017.
 */

import android.view.MenuItem;

public interface IMenuSettings {

    int[] getIdsOfMenuItemsToMakeVisible();

    void bindMenuItem(MenuItem item);
}

File: app/src/main/java/com/geridea/trentastico/gui/activities/IMenuSettings.kt
package com.geridea.trentastico.gui.activities


/*
 * Created with ♥ by Slava on 22/04/2017.
 */

import android.view.MenuItem

interface IMenuSettings {

    val idsOfMenuItemsToMakeVisible: IntArray

    fun bindMenuItem(item: MenuItem)
}
